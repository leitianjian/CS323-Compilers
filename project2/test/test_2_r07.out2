Error type 7 at Line 10: unsupported type of operands for arithmatic operator
Error type 5 at Line 10: unmatching type on both sides of assignment
