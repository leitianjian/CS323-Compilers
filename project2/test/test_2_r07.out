Error type 7 at line 10: unmatching operands
